deeplabcut/generate_training_dataset/multiple_individuals_trainingsetmanipulation.py

```diff
@@ -95,6 +95,7 @@ def format_multianimal_training_data(
     return train_data
 
 
+
```

Comment:
there should only be [2 blank lines](https://peps.python.org/pep-0008/#blank-lines) between functions :)

```suggestion
```